test/definitions/simple_workflow.json

```diff
@@ -6,6 +6,23 @@
   "version_minor": 0,
   "version_micro": 1,
   "tags": ["speech_to_text"],
+  "rights": {
+      "view": [
+          "group"
```

Comment:
add more relevant groups for unit testing !
like that you cannot test correctly multi-groups different rights etc.